setup.py

```diff
@@ -1,30 +1,37 @@
 from setuptools import setup, find_packages
 from pathlib import Path
 
-# Use semantic versioning
-VERSION = "1.0.0"
-
+# Minimum python version. Remember to also change in github workflows if increased
 PY_VERSION_REQUIRED = ">=3.7"
+
 PACKAGES_REQUIRED = {
     "core": [
         "numpy",
         "scipy",
     ],
     "test":[
-        "pytest"
+        "pytest",
     ]
 }
 
-# Read README.md to use as long description in package metadata
 root_dir = Path(__file__).parent.absolute()
+
+# Read README.md to use as long description in package metadata
 with (root_dir / "README.md").open("rt") as readme_file:
     long_description = readme_file.read().strip()
 
+# Read semantic version number from _version.py
+version_file_namespace = {} # type:ignore
+with (root_dir / "neffint" / "_version.py").open("r") as version_file:
+    exec(version_file.read(), version_file_namespace)
```

Comment:
Isn't there another (more pythonic...) way to do this? What about importing the version number, simply?

Reply:
what about something like (from [here](https://packaging.python.org/en/latest/guides/single-sourcing-package-version/) )

```
def get_version(rel_path):
    for line in read(rel_path).splitlines():
        if line.startswith('__version__'):
            delim = '"' if '"' in line else "'"
            return line.split(delim)[1]
    else:
        raise RuntimeError("Unable to find version string.")
```

and then using it with `version = get_version(os.path.join("neffint","_version.py"))`?